graylog2-web-interface/src/components/systemmessages/SystemMessagesComponent.jsx

```diff
@@ -82,9 +83,9 @@ class SystemMessagesComponent extends React.Component {
           <h2>System messages</h2>
 
           <p className="description">
-            System messages are generated by graylog-server nodes on certain events that may be interesting for the
-            Graylog administrators. You don&apos;t need to actively act upon any message in here because notifications
-            will be raised for any events that required action.
+            System messages are generated by <ProductName /> server nodes on certain events that may be interesting for
```

Comment:
In which cases are we going to use component <ProductName />, and in which just a value from the hook?

Reply:
I used the hook whenever possible, only in class components I implemented the component.